packages/react-composites/src/composites/common/icons.tsx

```diff
@@ -252,7 +256,11 @@ export type CallCompositeIcons = {
   /* @conditional-compile-remove(teams-meeting-conference) */
   PhoneNumberButton?: JSX.Element;
   /* @conditional-compile-remove(teams-meeting-conference) */
-  PhoneInfoWait?: JSX.Element;
+  JoinByPhoneDialStep?: JSX.Element;
```

Comment:
This says `Step` but others say `Icon` - not sure what the best name is here but can they be made consistent